src/locale/es-MX/dim.json

```diff
@@ -279,7 +279,7 @@
       "Vault": "There are too many '{{itemtype}}' items in the {{store}}."
     },
     "Classified": "This item is classified and cannot be transferred at this time.",
-    "Classified2": "Classified item. Bungie does not yet provide information about this item. Item is not yet transferable.",
+    "Classified2": "Classified item. Bungie does not yet provide information about this item. Add notes to this item and use the \"notes:\" search filter to find it.",
```

Comment:
Is there a way we can not include untranslated strings?

Reply:
I submitted a commit that shows what the "skip untranslated string" option does

Reply:
Hmm. My guess would be that that would result in empty strings, rather than fallback to English? Do you know what happens?

Ideally we'd just fallback to the English strings for anything untranslated (even better if we can fall back to es from es-mx).

Reply:
empty strings would occur, reverted the commit. Fallback would be ideal but would require manually editing each json file to remove the english strings. Fallback may be possible from es-mx to es. I will research that further.